lumen/ai/coordinator.py

```diff
@@ -364,21 +363,13 @@ async def _chat_invoke(self, contents: list | str, user: str, instance: ChatInte
         log_debug(f"New Message: \033[91m{contents!r}\033[0m", show_sep="above")
         await self.respond(contents)
 
-    @retry_llm_output()
```

Comment:
Not entirely sure this is necessary, and it was a bit complex to include the system prompt here with planner so I just removed.